README.rst

```diff
@@ -53,6 +53,8 @@ A docker package exists for this project::
 Then it can be used like::
 
   $ docker run  -v ./data:/data -v ./config:/config rss2email list
+  
```

Comment:
Whitespace on blank line.

Reply:
Will fix. Got a better setup now so that I won't miss these anymore